resource_customizations/platform.confluent.io/Connector/health.lua

```diff
@@ -0,0 +1,18 @@
+hs = {}
+if obj.status ~= nil and obj.status.state ~= nil then
+    if obj.status.state == "CREATED" and obj.status.connectorState == "RUNNING" then
+        hs.status = "Healthy"
+        hs.message = "Connector running"
+        return hs
+    end
+    if obj.status.state == "ERROR" then
+        hs.status = "Degraded"
+        for i, condition in ipairs(obj.status.conditions) do
+            hs.message = condition.message
```

Comment:
Would this cause the health check message to just match whatever is the last message in the conditions?

Reply:
Yeah it would in its current form. Open to suggestions of what would be preferred 😄

Reply:
I think it completely depends on the resource. The health check should take all relevant conditions into account.

Reply:
Hey @crenshaw-dev I've gone through and double checked a number of Connector failure scenarios with this Confluent Operator. Even though conditions is an array, it was only generating one condition each time. In addition, the message being returned in that condition encapsulates all the error messages within that condition. Therefore I've updated the logic and test scenario to mimic the scenario of what's happening. Let me know if any further changes are needed.